main.go

```diff
@@ -1,8 +1,47 @@
 package main
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func getPing(w http.ResponseWriter, r *http.Request) {
+	metodo := r.Method
+	fmt.Printf("metodo: %v\n", metodo)
+	if metodo == "GET" {
+		w.WriteHeader(http.StatusNoContent)
+	} else {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+	}
+
+}
+
+func getBooks(w http.ResponseWriter, r *http.Request) {
+
+	bookslist := []struct { //Struct created to handle data of the list of books
+		Name      string
+		Price     float32
+		Inventory int
```

Comment:
Check on how you can change the names of the properties in the JSON document by adding struct tags.
https://www.digitalocean.com/community/tutorials/how-to-use-struct-tags-in-go

When you have that figured out, change all the property name to lowercase.

Reply:
Done.